source/deprecations/v1.x.html.md

```diff
@@ -1172,6 +1172,68 @@ Object.freeze(c);
 c.swapNames();  // EXCEPTION
 ```
 
+#### Copyable.frozenCopy
+
+Just as the `Freezable` mixin is deprecated in favor of functionality in
+core JavaScript, the `frozenCopy` method of the Copyable mixin is also
+deprecated in favor of [`Object.freeze()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze).
+
+Replace the following code:
+
+```javascript
+const Obj = Ember.Object.extend(Freezable, Copyable, {
+  copy() {
+    return Obj.create();
+  }
+});
+
+const frozenCopy = Obj.create().frozenCopy();
+frozenCopy.isFrozen(); // => true
+frozenCopy.set('foo', 'baz'); // => throws TypeError
+```
+
+with:
+
+```javascript
+const a = Ember.Object.create();
+Object.isFrozen(a); // => false
+Object.freeze(a);
+Object.isFrozen(a); // => true
+a.set('foo', 'baz'); // => throws TypeError
+```
+
+#### Using `/` for namespace in the `{{render}}` helper
+
+Identifying controllers not under the application namespace no longer should take
+a slash as the separator, as Ember has settled on using dot notation for namespaces.
+
+Instead of using `{{render 'blog/posts'}}`, use `{{render 'blog.posts'}}`.
+
+#### Calling `Ember.set` and `Ember.get`
+
```

Comment:
How about "Non-Standard Use of Calling..."—we don't want to accidentally indicate that we've completely deprecated `Ember.set` and `Ember.get`.